ui/src/shared/components/Dygraph.js

```diff
@@ -341,64 +341,8 @@ export default class Dygraph extends Component {
     }
   }
 
-  highlightCallback = e => {
-    const chronografChromeSize = 60 // Width & Height of navigation page elements
-
-    // Move the Legend on hover
-    const graphRect = this.graphRef.getBoundingClientRect()
-    const legendRect = this.legendRef.getBoundingClientRect()
-
-    const graphWidth = graphRect.width + 32 // Factoring in padding from parent
-    const graphHeight = graphRect.height
-    const graphBottom = graphRect.bottom
-    const legendWidth = legendRect.width
-    const legendHeight = legendRect.height
-    const screenHeight = window.innerHeight
-    const legendMaxLeft = graphWidth - legendWidth / 2
-    const trueGraphX = e.pageX - graphRect.left
-
-    let legendLeft = trueGraphX
-
-    // Enforcing max & min legend offsets
-    if (trueGraphX < legendWidth / 2) {
-      legendLeft = legendWidth / 2
-    } else if (trueGraphX > legendMaxLeft) {
-      legendLeft = legendMaxLeft
-    }
-
-    // Disallow screen overflow of legend
-    const isLegendBottomClipped = graphBottom + legendHeight > screenHeight
-    const isLegendTopClipped =
-      legendHeight > graphRect.top - chronografChromeSize
-    const willLegendFitLeft = e.pageX - chronografChromeSize > legendWidth
-
-    let legendTop = graphHeight + 8
-    this.setState({legendArrowPosition: 'top'})
-
-    // If legend is only clipped on the bottom, position above graph
-    if (isLegendBottomClipped && !isLegendTopClipped) {
-      this.setState({legendArrowPosition: 'bottom'})
```

Comment:
Why do we no longer need to track `arrowPosition`?

Reply:
I was going to remove them and see if anyone noticed.  They frequently get out of position / sync with the hover line.

Reply:
K, fine by me